doc/workshop/exercises/analysis/clipping/clipping.rst

```diff
@@ -65,6 +65,17 @@ capability to reproject the data for us on-the-fly:
 .. image:: ../../../images/clipping-view-polygons.png
     :target: ../../../../_images/clipping-view-polygons.png
 
+.. note::
+    Notice the numbers on the buildings and trees. These are the classifations given in
+    the LIDAR Point Classes or `LAS Specification`_. You can sort and single out these in JSON filters.
+    ex. "limits": "Classification[3:4]" which only shows classes 3 to 4 which 
```

Comment:
This comment is relevant to a `filters.range` but not a `filters.expression`. It needs to be updated to be an expression.